samples/v1/types_config.go

```diff
@@ -64,6 +64,15 @@ type ConfigSpec struct {
 	// content but the operator will not recreate(or update) anything
 	// listed here.
 	SkippedTemplates []string `json:"skippedTemplates,omitempty" protobuf:"bytes,6,opt,name=skippedTemplates"`
+
+	// skippedhelmcharts specifies names of helm charts that should NOT be
+	// created/updated.  Admins can use this to allow them to delete content
+	// they don’t want.  They will still have to manually delete the
+	// content but the operator will not recreate(or update) anything
+	// listed here.
+	// +listType=set
+	// +kubebuilder:validation:MaxItems=16
+	SkippedHelmCharts []string `json:"skippedHelmCharts,omitempty" protobuf:"bytes,7,opt,name=skippedhelmCharts"`
```

Comment:
Is `Skipped` the best term here?

Reply:
it has to be kept inline with previous terms as skippedtemplates 
here there will be by default installtions given so for customer to tell the operatorr to skip managing it used the term skipped hence skipped is okay to be used

Reply:
> it has to be kept inline with previous terms as skippedtemplates

I would like to challenge that, why does it have to be kept inline? We are adding a new API, if there's a better name we could use, why not use it? We shouldn't propagate bad (I appreciate this is subjective) descriptions if we can think of better ones

Reply:
`Unmanaged` might be a better term but sticking w/ the existing prefix is has its own intrinsic value both for users who have already learned/understood the behavior of the other fields that share the prefix, as well as for new users trying to understand the set of fields as whole and how they relate.  This is my number one reservation about the "we don't double down on mistakes" philosophy.  Change+inconsistency have their own costs.

To use a new term for the new field should require not just that the new name be "better", but that it is sufficiently better to justify creating the inconsistency/confusion.  (It would also help if we had evidence the existing name has been a source of pain/confusion.  Personally i'm of the school of thought that no one uses an api field w/o reading the api docs for the field at which point the field name, as long as it's not explicitly wrong/misleading and as long as the api doc is decent, doesn't matter a whole lot)

Reply:
Definitely a fair point and I'm not saying Skipped is definitely a no go, I was kind of curious if we had any feedback, or perhaps, "I wish I'd called it X" moments. Opening up the discussion about potential other names. It feels to me like we should probably have some discussion about the name, even if at the end we come back to, no we had the best thing to begin with for XYZ reason, at least then we can justify the choice